import org.apache.hadoop.ha.FailoverFailedException;
public class FunctionUse3562 {
public void funcUse() {
FailoverFailedException failoverfailedexception = new FailoverFailedException();
}
}